tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestExampleGenerator.java

```diff
@@ -930,4 +944,133 @@ public void TestMinimalPairExamples() {
+                        if (!pluralOnly) {
+                            if (isVerbose()) {
+                                System.out.println();
+                            }
+                        }
+                    }
+                    if (path.startsWith(unused)) {
+                        pattern = "🚫  Not used with formatted units";
+                    } else {
+                        pattern = cldrFile.getStringValue(path);
+                        String actualRaw = exampleGenerator.getExampleHtml(path, pattern);
+                        examples = ExampleGenerator.simplify(actualRaw, false)
+                            .replace("〗〖", "\t")
+                            .replace("〗", "")
+                            .replace("〖", "")
+                            ;
+                        List<String> exampleList = com.google.common.base.Splitter.on('\t').trimResults().splitToList(examples);
+                        if (exampleList.size() != 2) {
+                            throw new IllegalArgumentException("Expecting exactly 2 examples: " + exampleList);
+                        }
+                        StringBuilder exampleBuffer = new StringBuilder();
+                        for (String exampleItem : exampleList) {
+                            if (exampleItem.contains("❬null❭") || exampleItem.contains("❬n/a❭")) {
+                                boolean bad = (exampleItem.contains("❌"));
+                                exampleItem = "🆖  No unit available";
+                                if (bad) {
+                                    exampleItem = "❌  " + exampleItem;
+                                }
+                            }
+                            if (exampleBuffer.length() != 0) {
+                                exampleBuffer.append('\t');
+                            }
+                            exampleBuffer.append(exampleItem);
+                        }
+                        examples = exampleBuffer.toString();
+                    }
+                    String line = (locale
+                        + "\t" + localeName
+                        + "\t" + label
+                        + "\t" + code
+                        + "\t" + pattern
+                        + "\t" + examples);
+                    if (pluralOnly) {
+                        pluralSheet.add(line);
+                    } else {
+                        if (isVerbose()) {
+                            System.out.println(line);
+                        }
+                    }
+                }
+            }
+            if (pluralOnly) {
+                pluralSheet.add("");
+            } else if (isVerbose()) {
+                System.out.println();
+            }
+        }
+        if (isVerbose()) {
+            System.out.println("#################### Plural Only ###################");
+            for (String line : pluralSheet) {
+                System.out.println(line);
+            }
+        }
+    }
```

Comment:
this new function is 125 lines, which is about ten times the recommended max for maintainable code :-)

Reply:
it's a good point, although this is test code and not general purpose code

Reply:
even test code should be maintainable :-)